deployment/arm/global.json

```diff
@@ -108,11 +105,34 @@
                             "routeConfiguration": {
                                 "@odata.type": "#Microsoft.Azure.FrontDoor.Models.FrontdoorForwardingConfiguration",
                                 "forwardingProtocol": "HttpsOnly",
+                                "customForwardingPath": "/auth",
```

Comment:
Just double checking, this is correct or should it be /.auth?

Reply:
When you enable app authentication on an app service it automatically sets up endpoints under the `/.auth` folder.  However - since we are using Frontdoor + APIM we need to also forwards requests to this folder. 

APIM doesn't support api paths that start with `.`  I'm not super happy about it so it goes from `/.auth` -> `/auth` -> `/.auth`.